db/job_history.go

```diff
@@ -2,24 +2,25 @@ package db
 
 import (
 	"github.com/flanksource/duty/models"
+	"github.com/flanksource/incident-commander/api"
 	"github.com/google/uuid"
 )
 
-func PersistJobHistory(h *models.JobHistory) error {
-	if Gorm == nil {
+func PersistJobHistory(ctx *api.Context, h *models.JobHistory) error {
+	if ctx.DB() == nil {
 		return nil
 	}
```

Comment:
I think we should return an error here, since we want to persist it but database is not there, not sure why this was `nil` before

Reply:
Makes sense. Although why do we even have to check for nil in this particular db function?
If db is nil, this func will never be called

Reply:
Can't really, I remember putting this because something was failing